plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/CassandraServer.java

```diff
@@ -57,27 +59,39 @@
 {
     private static Logger log = Logger.get(CassandraServer.class);
 
-    private static final int PORT = 9142;
+    private static final int DEFAULT_PORT = 9142;
```

Comment:
Can you try changing `native_transport_port` value to 9042 in `cu-cassandra.yaml` in a preparatory commit? Thus, we don't need to add `int port` argument to the constructor.